### PR TITLE
[Docs] Translated first chapter to Spanish

### PR DESCRIPTION
**Pull request by IvanitiX**
As part of issue #40, I translated the first chapter to Spanish. 
Hope this helps to improve the work on this white paper :)

### CODE REVIEW

**Comment by bpetetot** [comment]
I will ask someone to review it and merge it asap.
Thanks a lot 🙏  this is a great work!

File: es/01-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Capítulo 1 : Poniéndonos en marcha 🥚
+
+El mundo del Open Source (o Código Abierto) es vasto y puede ser fácil perderse en éste. Al empezar, este mundo puede parecer inaccesible e intimidante, por lo que una de las preguntas en este punto es: ¿Por dónde empiezo? ¿Qué reglas y códigos de conducta tengo que seguir? ¿Puedo contribuir si ni siquiera sé programar?
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:3]:
```suggestion
El mundo del Open Source (o Código Abierto) es vasto y puede ser fácil perderse en él. Al principio, puede parecer inaccesible e intimidante, por lo que una de las preguntas en este punto son: ¿Por dónde empiezo? ¿Qué reglas y códigos de conducta tengo que seguir? ¿Puedo contribuir si ni siquiera sé programar?

```

File: es/01-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Una primera idea, quizás la más simple, sería **identificar proyectos que te gusten, que suelas usar o que signifiquen mucho para ti.** Será más fácil empezar en un entorno familiar que te pueda mantener motivado a colaborar a largo plazo.
+
+También es importante tener en cuenta la madurez del proyecto y su envergadura.  Puede ser tentador empezar con frameworks grandes, librerías o herramientas como VS Code, MongoDB, Android, React…. Este tipo de proyectos tienen muchos colaboradores y muchos bugs por arreglar o características por añadir. Las contribuciones en este tipo de proyectos suelen dar algo de satisfacción cuando estas acaban siendo integradas; pero debido a que estos proyectos tienen millones de líneas de código, no son la mejor elección para empezar en el Open Source. Esas comunidades ya están bien desarrolladas y mucha gente se dedica a ofrecer contribuciones que sólo unos pocos mantenedores han de revisar. Por lo tanto, se puede tardar un tiempo antes de que tu contribución sea estudiada y potencialmente acabe siendo aceptada.
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:12]:
```suggestion
También es importante tener en cuenta la madurez del proyecto y su tamaño.  Puede ser tentador empezar con frameworks grandes, librerías o herramientas como VS Code, MongoDB, Android, React…. Este tipo de proyectos tienen muchos colaboradores y muchos bugs por arreglar o características por añadir. Las contribuciones en este tipo de proyectos suelen dar algo de satisfacción cuando estas acaban siendo integradas; pero debido a que estos proyectos tienen millones de líneas de código, no son la mejor elección para empezar en el Open Source. Esas comunidades ya están bien desarrolladas y mucha gente se dedica a ofrecer contribuciones que sólo unos pocos mantenedores han de revisar. Por lo tanto, se puede tardar un tiempo antes de que tu contribución sea estudiada y potencialmente acabe siendo aceptada.

```

File: es/01-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Capítulo 1 : Poniéndonos en marcha 🥚
+
+El mundo del Open Source (o Código Abierto) es vasto y puede ser fácil perderse en éste. Al empezar, este mundo puede parecer inaccesible e intimidante, por lo que una de las preguntas en este punto es: ¿Por dónde empiezo? ¿Qué reglas y códigos de conducta tengo que seguir? ¿Puedo contribuir si ni siquiera sé programar?
+No te preocupes, en este artículo os daremos todos los consejos que necesites para empezar con buen pie, elegir un buen proyecto para colaborar y así enviar tu primera contribución.
+
+## Primeros pasos 🐣
+
+Lo primero que habría que hacer es decidir en qué proyecto quieres empezar a hacer tu primera contribución. Teniendo en cuenta que hay millones de proyectos Open Source, puede ser difícil encontrar tal proyecto.
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:8]:
```suggestion
Lo primero que tendrias que hacer es decidir en qué proyecto quieres empezar a hacer tu primera contribución. Teniendo en cuenta que hay millones de proyectos Open Source, puede ser difícil encontrar tal proyecto.

```

File: es/01-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+**Únete a la comunidad**
+
+Un proyecto no sólo se limita a la documentación y el código fuente. Es también importante entender e interactuar con la comunidad del proyecto. Hay diferentes roles en un proyecto Open Source. Los tres principales:
+* Los mantenedores o colaboradores core, que están desde el principio del proyecto o han sido invitados a este rol gracias a todas las contribuciones que se han hecho.
+* Colaboradores, quienes participan en la vida del proyecto ofreciendo contribuciones (documentación, arreglos de bugs...), respondiendo preguntas o abriendo tickets.
+* Y los Usuarios, quienes usarán el proyecto y muchas veces darán feedback, mandan bugs, proponen mejoras o preguntan en foros.
+
+Las vías de comunicación en la comunidad varían según el proyecto. Puede ser una mailing list, un canal de Slack, un grupo de Discord u otra herramienta de este tipo. Ahí podrás hacer preguntas, ayudar a definir roadmaps, o simplemente discutir con otros.
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:55]:
```suggestion
Los medios de comunicación en la comunidad varían según el proyecto. Puede ser una mailing list, un canal de Slack, un grupo de Discord u otra herramienta de este tipo. Ahí podrás hacer preguntas, ayudar a definir roadmaps, o simplemente discutir con otros.

```

File: es/01-getting-started.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+## Tu primera contribución 🐥
+
+Ahora que has identificado un proyecto que te interese y hayas leído la documentación necesaria para empezar, estás listo para ofrecer tu primera contribución al mundillo.
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:59]:
```suggestion
Ahora que has identificado un proyecto que te interese y has leído la documentación necesaria para empezar, estás listo para ofrecer tu primera contribución al mundillo.

```

File: es/01-getting-started.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+**Programación**
+
+* Revisa los Códigos de Conducta;
+* Diseña y desarrolla tests;
+* Automatiza el proyecto (CI, CD...);
+* Arregla bugs;
+* Añade nuevas características.
+
+Si no sabes dónde empezar, consulta la lista de issues abiertos en el proyecto. En muchos de ellos, algunos tickets se identifican con etiquetas como *"Good First Issues"* o *"Beginner"*. Están identificadas por los colaboradores principales como contribuciones simples y tomas de contacto con el proyecto. También puedes echar un ojo a los tickets categorizados como *"Help needed/wanted"*, que esperan a que alguien pueda asignarse tal tarea.
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:95]:
```suggestion
Si no sabes por donde empezar, consulta la lista de issues abiertos en el proyecto. En muchos de ellos, algunos tickets se identifican con etiquetas como *"Good First Issues"* o *"Beginner"*. Están identificadas por los colaboradores principales como contribuciones simples y tomas de contacto con el proyecto. También puedes echar un ojo a los tickets categorizados como *"Help needed/wanted"*, que esperan a que alguien pueda asignarse tal tarea.

```

File: es/01-getting-started.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+Cuando hayas identificado una issue o un ticket al que deseas contribuir, deberías ubicarte en el contexto del proyecto. Así que no dudes en preguntar a otros colaboradores para pedir información. Las discusiones e intercambios de información son esenciales para empezar una nueva contribución. Te guiarán a través de las expectativas de la comunidad, cómo validar una solución técnica o identificar restricciones o dificultades potenciales.
+
+Antes de publicar tu primera contribución, comprueba tu código y no dudes en probarlo antes de enviarlo. No dudes tampoco en explicar el contexto y documentar tu contribución. Todo lo que ayude a otros colaboradores y mantenedores del proyecto para revisar la contribución harán acelerar ese proceso.
+
+Finalmente, cuando hayas hecho tu primera contribución, no olvides publicarlo y compartirlo con el mundo a través de Twitter, Facebook o tu red social de preferencia.
```
Comment (jeanphibaconnais) [inline es/01-getting-started.md:101]:
```suggestion
Finalmente, cuando hayas hecho tu primera contribución, no olvides publicarlo y compartirlo con el mundo a través de Twitter, Facebook o tu red social favorita.

```

**Review by jeanphibaconnais — COMMENTED** [review]
Hi @IvanitiX , sorry for the delay. I add comments on your PR. It's a great job, thanks 🙏

**Comment by IvanitiX** [comment]
Hi @jeanphibaconnais , I added all changes you commented. I have to say those changes will improve the readibility and comprehension of the text, so good catch! And thank you for reviewing the PR :)

**Comment by jeanphibaconnais** [comment]
Thanks you so much for your contribution @IvanitiX !

**Pull request merged**